sdk/canton/community/base/src/main/protobuf/com/digitalasset/canton/protocol/v5/participant_transaction.proto

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.digitalasset.canton.protocol.v5;
+
+import "com/digitalasset/canton/crypto/v0/crypto.proto";
+import "com/digitalasset/canton/protocol/v0/participant_transaction.proto";
+import "com/digitalasset/canton/protocol/v1/participant_transaction.proto";
+import "com/digitalasset/canton/protocol/v2/participant_transaction.proto";
+import "com/digitalasset/canton/protocol/v3/participant_transaction.proto";
+import "com/digitalasset/canton/protocol/v4/participant_transaction.proto";
+import "scalapb/scalapb.proto";
+
+// Compared to v4: action_description changed type from v3.ActionDescription to v4.ActionDescription
+message ViewParticipantData {
+  option (scalapb.message).companion_extends = "com.digitalasset.canton.version.StableProtoVersion";
```

Comment:
Should this have the stableprotoversion tag if its on PV7? @rgugliel-da 
Same goes for the V4 ActionDescription

Reply:
Including the `ActionDescription`, even though its defined in V4?